python/CMakeLists.txt

```diff
@@ -16,16 +16,14 @@ cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
 
 set(rmm_version 24.02.00)
 
-include(../fetch_rapids.cmake)
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-24.02/RAPIDS.cmake
```

Comment:
Why was this changed? We should continue to use `fetch_rapids.cmake` because it centralizes how we fetch/use rapids-cmake for both C++ and Python builds. However, if this must be changed for some reason, we'll need to be sure `update-version.sh` is updated correspondingly.

Reply:
This looks like a typo from testing. I'll revert.